arkane/pdep.py

```diff
@@ -107,6 +107,17 @@ def __init__(self, network,
                  maximumGrainSize=None, minimumGrainCount=0,
                  method=None, interpolationModel=None, maximumAtoms=None,
                  activeKRotor=True, activeJRotor=True, rmgmode=False, sensitivity_conditions=None):
+
+        if len(network.products) > 0:
+            if method in "simulation least squares":
```

Comment:
Why are we using in here?

Reply:
Sorry, should be flipped. We're looking to see if "simulation least squares" is within the method name.